### PR TITLE
feat: UwUśny feature

### PR DESCRIPTION
**Pull request by Sanielia**
- Added h3gg profile description to README
- Added polish and english version of XXV Anniversary Championships - Warmup tournament regulations and privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+We have organized many Heroes III tournaments i.e. XX Anniversary Tournament, World Championships, Polish Cup and XXV Anniversary Championships.
+
+## 📋 Our team:
+
+Our team consist of many expierenced Heroes III tournament organizers, players and template makers, ATM there are 20 of us:
+
+- Keszu - H3gg leader,
+- Berkas, CzileraInżyniera, Vuko & Zuuzel - our socialmedia admins,
+- Kicferk - H3gg website admin,
+- Kubaryt - H3gg Github profile admin,
```
Comment (Norbiros) [inline README.md]: Who is "kubaryt"?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+
+We are group of people, who love Heroes of Might and Magic III, and want to see it flourish.
+We have organized many Heroes III tournaments i.e. XX Anniversary Tournament, World Championships, Polish Cup and XXV Anniversary Championships.
+
+## 📋 Our team:
+
+Our team consist of many expierenced Heroes III tournament organizers, players and template makers, ATM there are 20 of us:
+
+- Keszu - H3gg leader,
+- Berkas, CzileraInżyniera, Vuko & Zuuzel - our socialmedia admins,
```
Comment (Norbiros) [inline README.md:20]: I don't know them

File: README.md [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+> [!WARNING]
+> Grzybobranie
 
-jedzcie dzieci!
\ No newline at end of file
+> [!IMPORTANT]
+> UwU UwU UwU UwU
+> ![](https://media.discordapp.net/attachments/1022538414328913930/1216040283406467162/image.png?ex=66082b44&is=65f5b644&hm=247cc5c49e003dc93d5f4ae2c556061ed35c96e7fecff1051641c66ceefa3451&=&format=webp&quality=lossless&width=550&height=310)
+
+# We are H3gg 👋
```
Comment (goteusz-maszyk) [inline README.md]: We're not.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,38 @@
...
+
+## 🐎 About us:
+
+We are group of people, who love Heroes of Might and Magic III, and want to see it flourish.
+We have organized many Heroes III tournaments i.e. XX Anniversary Tournament, World Championships, Polish Cup and XXV Anniversary Championships.
```
Comment (Norbiros) [inline README.md]: I never played heroses 3

**Review by goteusz-maszyk — CHANGES_REQUESTED** [review]

**Review by Norbiros — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Sanielia) [reply README.md]: Idk

File: README.md (hunk 3 above)
Comment (Sanielia) [reply README.md]: Okok

File: README.md (hunk 2 above)
Comment (Sanielia) [reply README.md:20]: Idc

File: README.md (hunk 4 above)
Comment (Sanielia) [reply README.md]: Idc

**Comment by goteusz-maszyk** [comment]
jednak muszę dać żebyś nie mógł zamykać konwersacji cwaniaczku

**Comment by tapczanek1321** [comment]
co czemu ja to mam reviewowac

File: README.md [hunk 5]
```diff
@@ -1,3 +1,38 @@
-# Grzybobranie
+> [!WARNING]
+> Grzybobranie
 
-jedzcie dzieci!
\ No newline at end of file
+> [!IMPORTANT]
+> UwU UwU UwU UwU
```
Comment (tapczanek1321) [inline README.md:5]: UwU

Comment (Sanielia) [reply README.md:5]: UwU

**Review by Norbiros — APPROVED** [review]
lgtm!

File: README.md [hunk 6]
```diff
@@ -1,3 +1,38 @@
...
+- Bejker, Fafhrd, KacperOK, Makiki, mvrcxlinv, Niechcic, Ortant, Ponczek, Ravciozo, Regrof, Sokol_53 and Valerin Bizonov, who also do other various things.
+
+## 🌐 Our website
+
+We have created biggest polish platform for organizing Heroes III tournaments.
+Including ongoing tournaments, on our website there are 84 tournaments, where most of them are organized by people outside our team.
+You can check it out https://www.h3.gg/.
+
+If you are interested in organizing Heroes III tournament, you can contact us and we will give you organizer permissions on our website.
+
```
Comment (goteusz-maszyk) [inline README.md:33]: that's not our website

File: README.md [hunk 7]
```diff
@@ -1,3 +1,38 @@
...
+## ✉️ How to reach us:
+
+- You can contact us on our [Discord server](https://discord.com/invite/WPXRs5aX89),
+- You can mail us: `support@h3.gg`
+- You also can write to us on our [Facebook](https://www.facebook.com/h3ggg/) or on our streams on [Twitch](https://twitch.tv/h3gg) and [Youtube](https://www.youtube.com/@H3ggtournaments)
```
Comment (goteusz-maszyk) [inline README.md:38]: that's not our contact

**Review by goteusz-maszyk — DISMISSED** [review]

File: README.md (hunk 6 above)
Comment (Sanielia) [reply README.md:33]: It is

File: README.md (hunk 7 above)
Comment (Sanielia) [reply README.md:38]: It is

**Pull request merged**